helmfile/upstream/falco-exporter/templates/prometheusrule.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if and .Values.prometheusRules.enabled .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "falco-exporter.fullname" . }}
+  {{- if .Values.prometheusRules.namespace }}
+  namespace: {{ .Values.prometheusRules.namespace }}
+  {{- end }}
+  labels:
+    {{- include "falco-exporter.labels" . | nindent 4 }}
+    {{- if .Values.prometheusRules.additionalLabels }}
+    {{- toYaml .Values.prometheusRules.additionalLabels | nindent 4 }}
+    {{- end }}
+spec:
+  groups:
+  - name: falco-exporter
```

Comment:
Do we already have all of these alerts or could some of these be worth adding?

Reply:
I assume this is a question for all reviewers

Reply:
seems that we do not have them we only have alerts when `falco_events != 0`
I would propose to create another issue to review the alert part after we finish all other falco improvement tasks.

Reply:
#1033